### PR TITLE
usability: cvm doubleclick behaviour

### PR DESCRIPTION
**Pull request by manavortex**
double click on collapsed element will collapse all of its siblings

### CODE REVIEW

File: WolvenKit/Views/Tools/RedTreeView.xaml.cs [hunk 1]
```diff
@@ -385,7 +381,14 @@ private void OnDoubleClick(object sender, MouseButtonEventArgs e)
                 return;
             }
 
+            // If the current chunk is not expanded, collapse all of its siblings
+            if (!chunk.IsExpanded && chunk.Parent is ChunkViewModel parent)
```
Comment (rfuzzo) [inline WolvenKit/Views/Tools/RedTreeView.xaml.cs:385]: I kinda don't understand this... so if the selected item is not expanded, then we select the parent?

**Comment by manavortex** [comment]
if the node is expanded and I double-click it, it'll expand/collapse its children
if the node is collapsed and I double-click it, it'll expand/collapse its siblings

**Comment by rfuzzo** [comment]
> if the node is expanded and I double-click it, it'll expand/collapse its children if the node is collapsed and I double-click it, it'll expand/collapse its siblings

🤔 hmm I do think I understand the concept, but how does the code work, it basically selects the parent correct?

**Comment by manavortex** [comment]
Yeah, the rest is already in. I just noticed today that I needed this after doing a lot of search&replace in an .app with 168 appearances and had to scroll up to the parent node each time to re-collapse the nodes

**Review by rfuzzo — APPROVED** [review]
ok

**Pull request merged**